src/dataflow/src/render/reduce.rs

```diff
@@ -342,7 +336,7 @@ where
                                             row_packer.push(datum);
                                             input = &input[1..];
                                         }
-                                        _ => log::error!("Aggregation erroneously reported as both accumulable and min/max in ReduceCollation"),
+                                        _ => log::error!("Aggregation erroneously reported as both accumulable and hierarchical in ReduceCollation"),
```

Comment:
update comment in accumulable_hierarchical fn to make it clear that we can't have them be both